prisma/schema.prisma

```diff
@@ -83,7 +83,7 @@ model Scope {
   name        String           @unique
   description String?          @db.Text
   color       String?
-  createdAt   DateTime         @default(now())
+  createdAt   DateTime         @unique @default(now())
```

Comment:
Why? (I suppose it's for sorting but my guess is to use multi column sorting and not this)

Reply:
Look at this code for reference https://github.com/ivan-dalmet/react-query-offline-demo/blob/master/pages/api/posts/index.ts 😉

Reply:
Yep, I'm thinking about adding a new field as the id is not autoincremented

Reply:
In the example the ID is a uuid, I don't think you need another field (or maybe I missing something)

Reply:
I changed for the `id` as the cursor. At the moment, I didn't change it to an integer as it seems to work. The documentation is quite evasive on the subjet.